styles/_base.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+html {
+  height: 100%;
+}
+fieldset {
+  padding: 0;
+  border: none;
+}
+button {
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+body {
```

Comment:
I would suggest that the styles for the body are directly after the universal selector.